### PR TITLE
Add ASE paper on PR descriptions

### PR DESCRIPTION
**Pull request by kanghj**
Signed-off-by: Kang Hong Jin <kanghongjin@gmail.com>

Please fill the `[ ]` with an `x` to make sure that you read and followed the checklist

- [x] This addition is about Machine Learning __on Code__ specifically. General links should be added to [Awesome Machine Learning](https://github.com/josephmisiti/awesome-machine-learning) instead.
- [x] I have checked that the proposed addition is not already in the awesome list.
- [x] I have signed-off my commits as detailed [here](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md).
- [x] The links follow the link template:

        * [Title](URL) - Description.
    or

        * [Title](URL) - Description [Code](link to code).
- [x] The addition is correctly classified as a paper, blog post, talk, software or dataset.
- [x] The commit message must start with a verb (`Add …`, `Deprecate …`) and end without a dot. For exemple:

        Deprecate Theano software

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -253,6 +253,7 @@ A curated list of awesome research papers, datasets and software projects devote
 - <img src="badges/11-pages-gray.svg" alt="11-pages" align="top"> [A Convolutional Attention Network for Extreme Summarization of Source Code](https://arxiv.org/abs/1602.03001) - Miltiadis Allamanis, Hao Peng, Charles Sutton, ICML 2016.
 - <img src="badges/4-pages-gray.svg" alt="4-pages" align="top"> [TASSAL: Autofolding for Source Code Summarization](http://homepages.inf.ed.ac.uk/csutton/publications/icse2016-demo.pdf) - Jaroslav Fowkes, Pankajan Chanthirasegaran, Razvan Ranca, Miltiadis Allamanis, Mirella Lapata, Charles Sutton, ICSE 2016.
 - <img src="badges/11-pages-gray.svg" alt="11-pages" align="top"> [Summarizing Source Code using a Neural Attention Model](https://github.com/sriniiyer/codenn/blob/master/summarizing_source_code.pdf) - Srinivasan Iyer, Ioannis Konstas, Alvin Cheung, Luke Zettlemoyer, ACL 2016.
+- <img src="badges/13-pages-gray.svg" alt="11-pages" align="top"> [Automatic Generation of Pull Request Descriptions](https://arxiv.org/abs/1909.06987) - Zhongxin Liu, Xin Xia, Christoph Treude, David Lo, Shanping Li, ASE 2019.
```
Comment (m09) [inline README.md]:
```suggestion
- <img src="badges/13-pages-gray.svg" alt="13-pages" align="top"> [Automatic Generation of Pull Request Descriptions](https://arxiv.org/abs/1909.06987) - Zhongxin Liu, Xin Xia, Christoph Treude, David Lo, Shanping Li, ASE 2019.
```

**Review by m09 — CHANGES_REQUESTED** [review]
Thank you @kanghj for the interesting paper! A small edit and we're good to go.

File: README.md (hunk 1 above)
Comment (kanghj) [reply README.md]: Nice catch! I'll amend the commit

**Review by m09 — APPROVED** [review]

**Comment by m09** [comment]
Thanks for the contribution!

**Pull request merged**